posts/db/migrate/20230830020829_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :content
+      t.integer :upvotes, default: 0
+      t.integer :downvotes, default: 0
+      t.belongs_to :user, index: true
```

Comment:
O user não deveria estar associado ao voto em vez do post?

Reply:
Fiz a associação pensando na lógica "Auhor/Post" que vemos em tantos challenges, para que cada post tenha seus "créditos". Para associar user com vote, precisaríamos da associação many-to-many (como comentei anteriormente)